Xamarin.Forms.Material.iOS/MaterialFrameRenderer.cs

```diff
@@ -204,6 +203,8 @@ void UpdateBorderColor()
 					colorScheme.OnSurfaceColor = _defaultCardScheme.ColorScheme.OnSurfaceColor;
 				else
 					colorScheme.OnSurfaceColor = borderColor.ToUIColor();
+
+				this.SetBorderWidth(1f, UIControlState.Normal);
```

Comment:
I think that it is necessary to set 1 only there is a color not by default.
```C#
	SetBorderWidth(borderColor.IsDefault ? 0f : 1f, UIControlState.Normal);
```
And why `1`? In implementation for android, it is `3`